mteb/overview.py

```diff
@@ -288,6 +297,7 @@ def get_task(
         languages: A list of languages either specified as 3 letter languages codes (ISO 639-3, e.g. "eng") or as script languages codes e.g.
             "eng-Latn". For multilingual tasks this will also remove languages that are not in the specified list.
         script: A list of script codes (ISO 15924 codes). If None, all scripts are included. For multilingual tasks this will also remove scripts
+        eval_splits: A list of evaluation splits to include. If None, all splits are included.
```

Comment:
Could we also add a test case for the 2nd sentence, where we assert the number of splits > 1? (If none...)

Reply:
In that case, it should be: 
- task.eval_splits == task.metadata.eval_splits # can be 1
I have added a test for that.